### PR TITLE
Add about section

### PR DESCRIPTION
**Pull request by lukewestby**
Adds a component for the about section on the index page and implements a little bit of paragraph formatting to keep things looking consistent.

### CODE REVIEW

File: gatsby-site/src/components/About.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  return (
+    <Section>
+      <Heading>About Sunrise {hubName}</Heading>
+      <Body>
+        {content
+          .split(/\n\n+/)
+          .filter(entry => entry)
+          .map((entry, index) => (
+            <Paragraph key={index}>{entry}</Paragraph>
+          ))}
```
Comment (asross) [inline gatsby-site/src/components/About.js:54]: This seems like a good solution for now, but I want to mention that we could allow markdown / some kind of WYSIWYG functionality for the about section content, if we want.

File: gatsby-site/src/components/About.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+`
+
+/**
+ * @typedef {Object} AboutProps
+ * @property {string} content
+ * @property {string} hubName
+ *
+ * @param {AboutProps} props
+ */
+const About = ({ hubName, content }) => {
```
Comment (nehahaa) [inline gatsby-site/src/components/About.js:44]: What are the pros/cons of const vs. function?

Comment (faarwa) [inline gatsby-site/src/components/About.js:44]: What're the pros/cons of const vs function?

**Review by faarwa — APPROVED** [review]
lgtm! was there a type system added?

File: gatsby-site/src/components/About.js (hunk 2 above)
Comment (lukewestby) [reply gatsby-site/src/components/About.js:44]: In the context of a react component the differences between const and function don't really matter. const is just my habit. If you'd prefer to use function I'd be happy to change it.

**Comment by lukewestby** [comment]
> was there a type system added?

@faarwa the typescript typechecker built into VS Code will check javascript code based on JSDoc comments. They're just regular javascript comments and I didn't add any additional tools to the repo but if your editor knows what to do you can get type checking anyway.

**Pull request merged**